src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelGene.tsx

```diff
@@ -68,18 +67,55 @@ const TrackPanelGene = (props: TrackPanelGeneProps) => {
       trackId: GENE_TRACK_ID
     })
   );
-  const { toggleTrack, genomeBrowser } = useGenomeBrowser();
-
+  const browserGenomeTrackStates = useAppSelector(
+    getBrowserActiveGenomeTrackStates
+  );
+  const { toggleTrack, updateTranscriptTracks, genomeBrowser } =
```

Comment:
> `updateTranscriptTracks`

Wondering about the name here. On Dan's side, there are no transcript tracks. Are we keeping the metaphor on our side, or do we want to change the names (e.g. to focusGeneTranscripts) to keep consistency with Dan's view of the world?